generated/nifake/nifake/unit_tests/test_converters.py

```diff
@@ -294,6 +294,92 @@ def test_repeated_capabilities_invalid_resource_names():
     assert test_result_list == '0/1:2,'
 
 
+def test_convert_single_group_repeated_capabilities():
+    test_result_list = _converters.convert_single_group_repeated_capabilities('0')
+    assert test_result_list == ['0']
+    test_result_list = _converters.convert_single_group_repeated_capabilities('0-2')
+    assert test_result_list == ['0', '1', '2']
+    test_result_list = _converters.convert_single_group_repeated_capabilities('0:2')
+    assert test_result_list == ['0', '1', '2']
+    test_result_list = _converters.convert_single_group_repeated_capabilities('Dev1/0')
+    assert test_result_list == ['Dev1/0']
+    test_result_list = _converters.convert_single_group_repeated_capabilities('Dev1/0-2')
+    assert test_result_list == ['Dev1/0', 'Dev1/1', 'Dev1/2']
+    test_result_list = _converters.convert_single_group_repeated_capabilities('Dev1/0:2')
+    assert test_result_list == ['Dev1/0', 'Dev1/1', 'Dev1/2']
+
+
+def test_convert_channels_repeated_capabilities_without_prefix():
```

Comment:
Instead of "with/without prefix" use term "fully_qualified_channel_name" and "not_fully_qualified_channel_name".
This is a widely used concept by MI driver developers and public driver documentation to distinguish "0" from "Dev1/0".

Reply:
The new unit tests are now named `test_expand_channel_string_non_fully_qualified_channel_names()` and `test_expand_channel_string_fully_qualified_channel_names()` respectively.